media/css/externalpages/pocket/components/pocket-features.scss

```diff
@@ -0,0 +1,237 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/split';
+@import '~@mozilla-protocol/core/protocol/css/base/elements/containers';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-family';
+
+@import '../utils/variables';
+@import '../includes/nav';
+@import '../includes/footer';
+
+.mzp-c-split {
+    padding: 64px 0;
+}
```

Comment:
**Suggestion (non-blocking)**: Not sure if this is an issue or not, but since this would be affecting all split components, would it be better to make this more specific? Like `pocket-feature.mzp-c-split`?

Reply:
Hmm, I thought that because this CSS file is only being used for the pocket-features pages, then the style written in this file would only affect those HTML pages, no?

Reply:
I'm not sure! I was always just wary of it. @craigcook, any advice here?

Reply:
Correct, this rule applies to all split components on any page that references this style sheet. But this is a high-level component class that appears in lots of places, so even if it's just in this file on just this page, it might still be better to make the rule a little more specific to just these few instances of the component.

I guess my bigger question is: why does this component need to be modified here in the first place? Could the standard component work without changing it? If it has to be changed, maybe it's not the right component?

We should try to avoid modifying or overriding what comes from Protocol. It's not always possible and we'll inevitably make tweaks here and there, but every one of those little tweaks adds a little more technical debt and potential for something breaking if/when Protocol changes upstream. It eventually starts to undermine the point of standardized components if every page has a slightly different version.

I know the Pocket pages are kind of a special circumstance right now, and maybe this will be temporary and we'll be able to rebuild these pages soon, but the reality is they'll probably be around for a long time. We should make sure we're minimizing future headaches.

Also, I think 64px could be a `$layout-lg` token.

Reply:
Good insight, @craigcook thanks for explaining! 

I implemented the rule because there's a good chunk of top and bottom padding for each split component and some of the Pocket pages use multiple Split components sequentially, creating a lot more space in the page compared to the original pages. At first I decided to not tinker with the spacing, but then I really started noticing the differences, and decided to do something about them. 

I think the best course of action to lower the padding between each Split would be to give them a unique classname that specifically target the Split components I want manipulated (i.e. `pocket-feature-split`). What do you think?

Reply:
> I think the best course of action to lower the padding between each Split would be to give them a unique classname that specifically target the Split components I want manipulated (i.e. `pocket-feature-split`). What do you think?

Agreed, that's probably best. Giving them a different class is a more future-proof way to extend the component. It says "this is still a Split, but it's a special variant of a Split" instead of changing the base component.